tests/queries/0_stateless/02074_http_flush_compressed_buffers_on_cancel.reference

```diff
@@ -0,0 +1,2 @@
+zstd OK
```

Comment:
@tavplubix FYI: Currently it also broken on other compressions like `brotli`

for example
```
echo "select throwIf(number > 10000000) + number check from numbers(10000005) format CSV" | curl "http://127.0.0.1:8123/?query=&enable_http_compression=1" -H "Accept-Encoding: br" --compressed -v --data-binary @- | tail -c 1024
```

is the intention is to fix all the broken compression in this PR or just gzip?

(it would be nice to write tests for [all the compression supported](https://clickhouse.com/docs/interfaces/http#compression) by CH server?)

Reply:
Thanks @tavplubix. make sense :+1: